src/code/index.js

```diff
@@ -88,4 +88,38 @@ function previewLink(button) {
     console.log("Placeholder value:", inputElement.placeholder);
     console.log("Input value:", inputValue);
     // Rest of the code...
-}
\ No newline at end of file
+}
+
+//Function to check if a string is a valid URL
+function isValidURL(url) {
+    try {
+        new URL(url);
+        return true;
+    } catch (error) {
+        return false;
+    }
+}
```

Comment:
I have a doubt :thinking: 
Is it handling all types of URLs ?
Can you show some examples with the output of `isValidURL()` function for handling all the URLs specified in issue #41